rust_playground_highlight.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license       = "MIT"
 
-  gem.add_dependency "jekyll", ">= 2.0", "< 4.0"
+  gem.add_dependency "jekyll", ">= 2.0", "<= 4.2"
```

Comment:
Does this still work under Jekyll 2? Or should this bump the minimum Jekyll version?

Reply:
Good idea. I updated the code